### DISCUSSION TITLE
Permissions on file level

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#222] [Q&A] [answered]
Hello,
can we use the mask feature to assign permissions to files? can we do it in another way ?
I want users to be able to delete some files in a folder but not the entire folder.
Please advise.
Thanks.

**Comment by GeorgeGeorgeGe** [comment]
Sorry, I have just found out how to do that with the great feature: "Mask".

**Comment by rejetto** [comment]
I'm glad that you've found.
Please show me your solution.
It can be useful for me to see how people is using masks, as there is still much work to do on it for me.

**Comment by GeorgeGeorgeGe** [comment]
in the admin panel >> shared Files i selected the folder and add the following to the mask:
"*.txt":
  can_delete: false
this prevented any txt file in the folder from being deleted.

**Reply by rejetto** [reply, marked as answer]
nice, thanks.
One day I will make an easier editor.
